components/Header.js

```diff
@@ -1,21 +1,92 @@
 import Link from 'next/link'
+import { useState, useRef } from 'react';
+
 import styles from '../styles/Header.module.css';
+import useOnClickOutside from '../hooks/useOnClickOutside.js';
 
 export default function Header({ activePage }) {
+    const [isOpen, setIsOpen] = useState(false);
+    const [isOpenSearch, setIsOpenSearch] = useState(false);
+    const mobileNavMenuRef = useRef(null);
+    const mobileNavSearchRef = useRef(null);
+    useOnClickOutside(mobileNavMenuRef, () => setIsOpen(false));
+    useOnClickOutside(mobileNavSearchRef, () => setIsOpenSearch(false));
+
     return (
-        <header>
-           Header
-           
-           <Link href="/">
-                <a className={`${styles.link} ${activePage === 'Home' ?  styles.activeLink : '' }`}>
-                    Home
-                </a>
-            </Link>
-            <Link href="/products">
-                <a className={`${styles.link} ${activePage === 'Products' ?  styles.activeLink : '' }`}>
-                    Products
-                </a>
-            </Link>
-        </header>
+        <React.Fragment>
+
+            <meta name="viewport" content="width=device-width, initial-scale=1"></meta>
+
+            <nav className={styles.navBar}>
+
+                <button
+                    className={styles.menuIcon}
+                    onClick={() => setIsOpen(!isOpen)}>
+                </button>
+
+                <img src="/HMLogo.svg" alt="logo" className={styles.logo} href="/" />
```

Comment:
Should wrap this in a `<Link>` and link to the homepage. Maybe we should remove the Home from the nav links since this will link there? Not sure this is more of a design choice.

Reply:
I see a lot of websites that do both. It really "dumifies" it down. Because we would know that the logo would redirect to the home page but some people don't. What do you think, should we just get rid of Home?